expando.user.js

```diff
@@ -173,6 +189,13 @@
                 }
                 var images = mutation.addedNodes[j].querySelectorAll('img');
                 for (var k = 0; k < images.length; k++) {
+                    (function (image) {
```

Comment:
You should just move the whole thing to `for (const image of images) {` and get rid of the closure.

Reply:
@MadaraUchiha Hmm, that would work. But would that be supported in enough browsers?

Reply:
Supported in latest chrome and latest firefox. The rest can go duck themselves.

Reply:
@MadaraUchiha Hmm, in that case I'm going to refactor (?) it if that's okay with you